src/styles/reset.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  list-style: none;
+  text-decoration: none;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
```

Comment:
don't reset styles for *, set it for tags directly

only box-sizing leave here

font-family set for html or body